packages/ember-data/lib/adapters/rest-adapter.js

```diff
@@ -362,13 +362,15 @@ export default Adapter.extend(BuildURLMixin, {
     @return {Promise} promise
   */
   findAll: function(store, type, sinceToken) {
-    var query;
+    var query, url;
 
     if (sinceToken) {
       query = { since: sinceToken };
     }
 
-    return this.ajax(this.buildURL(type.typeKey), 'GET', { data: query });
+    url = this.buildURL(type.typeKey, undefined, undefined, 'findAll');
```

Comment:
Could these missing arguments be `null` instead of `undefined`? This will make it more consistent with https://github.com/emberjs/data/pull/2898/files#diff-24089ab4caa677e410cbc5c13588c012L533